scripts/model_gen_ng/model_gen_ng/templates/struct.tpl

```diff
@@ -0,0 +1,93 @@
+[[ struct.preamble ]]
+
+[%- for header in struct.includes %]
+#include "[[ header ]]"
+[% endfor %]
+
+[[ struct.doc | docstring -]]
+struct [[ struct.fullname ]]
+        [%- if struct.inherits %] : public [[ struct.inherits.user_fullname ]] [% endif -%]
+        {
+    [#- --- Member list --- #]
+    [%- for field in struct.fields %]
+    [[ field.doc | docstring ]]
```

Comment:
Don't we need `///` here?

Reply:
`docstring` takes care of that, also because `///` has to be injected for each line of the doc